spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binding/BindableProxyFactory.java

```diff
@@ -192,7 +192,7 @@ private boolean isPollable(Class<?> channelType) {
 	}
 
 	private void bridgeSubscribableToPollableChannel(SubscribableChannel sharedChannel, MessageChannel inputChannel) {
```

Comment:
maybe these param names should be: `subscribableChannel` and `pollableChannel` - also could we use the `PollableChannel` type for the latter?